docs/sources/installation/fedora.md

```diff
@@ -42,6 +48,12 @@ To update the `docker` package:
 
 Please continue with the [Starting the Docker daemon](#starting-the-docker-daemon).
 
+### Uninstall Docker
+
+To uninstall the `docker` package:
```

Comment:
Agreed, I prefer "Docker package" unless we are literally referring to a typed command or file/executable.